### DISCUSSION TITLE
Possible issue after Livewire(including Alpine) upgrade to V3

### DISCUSSION THREAD

**Discussion by colinmackinlay** [ryangjchandler/alpine-clipboard#36] [Q&A] [answered]
Hello - has anyone had any issues after upgrading to Livewire v3 with Alpine built in?

Previously I had the following working fine (with the package installed through npm):
```
                <button x-clipboard.raw="{{ $clipboard }}" wire:click="copied">
                    <i class="fad fa-copy pr-2"></i>
                </button>
```
SInce upgrading though I get this error in my console:
`Uncaught TypeError: Cannot read properties of undefined (reading 'writeText')`

It also happens if I omit from the npm build and just pull in as the CDN through `@assets`

**Comment by ryangjchandler** [comment, marked as answer]
This error is normally related to the fact that the code is running in an HTTP environment, not HTTPS. Are you using Valet? If so, try `valet secure`.

**Reply by colinmackinlay** [reply]
@ryangjchandler thank you for such a swift respopnse. I'd just tracked this down myself and it's not related to your excellent package or to Livewire. 

I use Laravel Sail so the issue is that although it's being served from "localhost" i.e. 127.0.0.1 the url for the app isn't localhost it's `laravel.test` as well as `subdomain.laravel.test` so the browser determines whether it is secure from the protocol. 

Sail serves with http rather than https which is where it is failing. It works fine in production!
